database/seeders/TypeSeeder.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class TypeSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+        $types= ['Sea-view', 'Pool-view', 'Garden-view', 'Penthouse','Mountain-view'];
+        foreach ($types as $type)
+        {
```

Comment:
{ on the same line as foreach please - we only put the bracket on the next line when it's a method in a class